docs/get-details/asa.md

```diff
@@ -64,7 +64,7 @@ Specifying a reserve account signifies that non-minted assets will reside in tha
 
 [**Freeze Address**](../transactions/transactions#freezeaddr)
 
-The freeze account is allowed to freeze or unfreeze the asset holdings for a specific account. When an account is frozen it cannot send or receive the frozen asset. In traditional finance, freezing assets may be performed to restrict liquidation of company stock, to investigate suspected criminal activity or to deny-list certain accounts. If the DefaultFrozen state is set to True, you can use the unfreeze action to authorize certain accounts to trade the asset (such as after passing KYC/AML checks). 
+The freeze account is allowed to freeze or unfreeze the asset holdings for a specific account. When an account is frozen it cannot send or receive the frozen asset. In traditional finance, freezing assets may be performed to restrict liquidation of company stock, to investigate suspected criminal activity or to deny-list certain accounts. If the DefaultFrozen state is set to True, you can use the unfreeze action to authorize certain accounts to trade the asset (such as after passing KYC/AML checks). Just remember that DefaultFrozen is an [immutable parameter](#immutable-asset-parameters) and cannot be changed after creation.
```

Comment:
Can you make this a !!note or something

Reply:
Will do. There was a `!!! tip` directly below this paragraph which made me tag it on to the end. But I'll put it in a `!!! note` and if it looks weird look at doing something else.